keras/applications/regnet.py

```diff
@@ -0,0 +1,1349 @@
+# Copyright 2021 Aditya Kane. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+# pylint: disable=invalid-name
+# pylint: disable=missing-docstring
+"""RegNet models for Keras.
+
+References:
+  - [Designing Network Design Spaces](https://arxiv.org/abs/2003.13678)
+    (CVPR 2020)
+  - [Fast and Accurate Model Scaling](https://arxiv.org/abs/2103.06877)
+    (CVPR 2021)
+"""
+
+from keras.applications.efficientnet import BASE_DOCSTRING, BASE_WEIGHTS_PATH
```

Comment:
This import statement is not needed since these variables are redefined in the file.